From 65243fc4f58c040f12b87df4d6d74001be8110e2 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 14 Apr 2018 14:41:20 +0300
Subject: [PATCH 4/6] Revert "sepolicy: suppress denial logspam"

* A more complete fix is pulled from AOSP master
  in system/sepolicy

This reverts commit 6fcb0af8f33b3b9243ddd4e5a7cf6aff47469882.

Change-Id: I6884a90cba11725add1ab6298baa92e20ce88893
---
 common/private/priv_app.te | 2 --
 1 file changed, 2 deletions(-)
 delete mode 100644 common/private/priv_app.te

diff --git a/common/private/priv_app.te b/common/private/priv_app.te
deleted file mode 100644
index 52073d7..0000000
--- a/common/private/priv_app.te
+++ /dev/null
@@ -1,2 +0,0 @@
-dontaudit priv_app device:dir read;
-dontaudit priv_app proc_interrupts:file read;
-- 
2.17.0

